app/models/dialog.rb

```diff
@@ -182,7 +182,7 @@ def dialog_field_hash
 
   def reject_if_has_resource_actions
     if resource_actions.length > 0
-      raise _("Dialog cannot be deleted because it is connected to other components.")
+      raise _("Dialog cannot be deleted because it is connected to other components: #{resource_actions.each.pluck(:resource_id, :resource_type).flatten}")
```

Comment:
The `.each` seems to be unused.

Would `resource_actions.pluck(:resource_type, :resource_id)` provide a better error message?

Reply:
Yes, if you'll let me say this should be backported to g and h.